File: Python/Toko.py
from Stok import Stok
from Penjualan import Penjualan

class Toko(Stok, Penjualan):
    def __init__(self, nama_toko="", alamat="", jumlah=0, tanggal="", harga=0, komputer=None):
        Stok.__init__(self, jumlah, komputer)
        Penjualan.__init__(self, tanggal, harga, komputer)
        self.nama_toko = nama_toko
        self.alamat = alamat

    def set_nama_toko(self, nama_toko):
        self.nama_toko = nama_toko

    def set_alamat(self, alamat):
        self.alamat = alamat

    def get_nama_toko(self):
        return self.nama_toko

    def get_alamat(self):
        return self.alamat

File: CPP/Toko.cpp
#pragma once
#include <iostream>
#include <string>
#include <vector>
#include "Stok.cpp"
#include "Penjualan.cpp"

using namespace std;

class Toko : public Stok, public Penjualan
{
private:
    string namaToko;
    string alamat;
public:
    Toko()
    {


    }


    Toko(string namaToko, string alamat, int jumlah, string tanggal, int harga, Komputer komputer) : Stok(jumlah, komputer), Penjualan(tanggal, harga, komputer)
    {
        this->namaToko = namaToko;
        this->alamat = alamat;
    }


    void setNamaToko(string namaToko)
    {
        this->namaToko = namaToko;
    }

    void setAlamat(string alamat)
    {
        this->alamat = alamat;
    }

    string getNamaToko()
    {
        return this->namaToko;
    }


    string getAlamat()
    {
        return this->alamat;
    }

    ~Toko()
    {


    }
};
